### PR TITLE
Installation: Referencing 'global' xmipp.conf instead of using local copy of it

### PR DESCRIPTION
**Pull request by DStrelak**
See also:
https://github.com/I2PC/xmipp/pull/487
https://github.com/I2PC/xmippViz/pull/53

### CODE REVIEW

**Review by fonsecareyna82 — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -77,8 +77,8 @@ jobs:
       # Build only the core to be able to run the static code analysis
       - name: Compile XmippCore
         run: |
-          cp ${BUILD_DIR}/xmipp.conf ./install
-          build-wrapper-linux-x86-64 --out-dir bw-outputs scons . -j4
+          cp ${BUILD_DIR}/xmipp.conf ./
+          build-wrapper-linux-x86-64 --out-dir bw-outputs /usr/bin/env python3 $(which scons) . -j4
```
Comment (albertmena) [inline .github/workflows/main.yml:81]: works with python 2.7 out of scipion3 env?

Comment (DStrelak) [reply .github/workflows/main.yml:81]: No. Python3 is required (also for scipion, so it should be there)

Comment (DStrelak) [reply .github/workflows/main.yml:81]: Also notice that this change is only for the github action script

Comment (albertmena) [reply .github/workflows/main.yml:81]: ok, thanks

**Review by albertmena — APPROVED** [review]

**Pull request merged**